srdg/holesky-testnet/application.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: holesky-testnet
+  namespace: argocd
+spec:
+  project: default
+  source:
+    repoURL: https://github.com/planetarium/9c-infra.git
+    targetRevision: feature/static-peer
```

Comment:
i wont merge this PR before test succeed, i will consider security after p2p goes well
@eseiker